gemcalibration/include/gem/calib/Calibration.h

```diff
@@ -20,7 +20,7 @@
 namespace gem {
   namespace calib {
 
-    enum calType {NDEF, PHASE, LATENCY, SCURVE, SBITRATE, THRESHOLD, TRIMDAC, DACSCANV3, TEMPERATURE, SBITREADOUT, SBITMAPANDRATE}; 
+    enum calType {NDEF, PHASE, LATENCY, SCURVE, SBITRATE, THRESHOLD, TRIMDAC, DACSCANV3, TEMPERATURE, SBITREADOUT, SBITMAPANDRATE, CALIBRATEARMDAC}; 
```

Comment:
Ups, I guess my comment to this line got lost when I changed the base branch...
Please remove the TRESHOLD as it is the same as CALIBRATEARMDAC, remove the TEMPERATURE, SBITREADOUT and SBITMAPANDRATE as non-calibration tools. Rename the DACSCANV3 to ARMDACSCAN as per @dorney comment

Reply:
Hello Misha, 
I removed the TEMPERATURE monitoring.
SBITREADOUT and SBITMAPANDRATE are still in there, but I can remove them right away if you confirm.  

For what concerns the  THRESHOLD, following Brian's comment, I changed it to ARMDACSCAN and looks like this atm.
<img width="1246" alt="Screen Shot 2019-03-28 at 4 39 07 PM" src="https://user-images.githubusercontent.com/6152470/55172829-f7fe2a00-517a-11e9-875b-2e2e1c97fdd8.png">
The CALIBRATEARMDAC in my understanding is another routine that had to be implemented and atm looks like, but should be modified  (Triggertype removed and list of ARM DAC settings to scan implemented)
<img width="1043" alt="Screen Shot 2019-03-28 at 4 22 33 PM" src="https://user-images.githubusercontent.com/6152470/55172994-3bf12f00-517b-11e9-8ac3-b6896d3d3590.png">
 DACSCANV3 is also a different one atm and looks like this:
<img width="1255" alt="Screen Shot 2019-03-28 at 4 56 52 PM" src="https://user-images.githubusercontent.com/6152470/55173230-9c806c00-517b-11e9-9ae6-44cc9e0e818c.png">

Please let me know if this is incorrect and which differences  are/aren't sopposed to be between these calibrations. 

Thanks and best, 
Camilla

Reply:
Hi @mexanick , I removed also the SBITREADOUT and SBITMAPANDRATE, following your suggestion. Please mind that in my last PR the other Calibration routines discussed above are still there. 
Thanks and cheers,
Camilla